### DISCUSSION TITLE
Remove player from Groups endpoint

### DISCUSSION THREAD

**Discussion by tim140701** [isFakeAccount/psnawp#51] [Ideas]
Hey, apologies if this in the wrong section - I've been really looking into the psn api. I saw all of your hard work and research on it and was wondering if you ever found an endpoint for kicking players from a group. Would be really helpful as wish to make a discord bot to control a psn group that usually stays at 100/100 players and sometimes needs people removed to add others.

**Comment by isFakeAccount** [comment]
I have to look into the end point for kicking people out.

**Reply by tim140701** [reply]
how do you find endpoints? I tried listening to network traffic w ssl certificate on my phone and using the psn app but wasn’t able to log in.

**Reply by isFakeAccount** [reply]
I followed this tutorial. https://httptoolkit.com/blog/frida-certificate-pinning/

You need rooted android phone.

**Reply by isFakeAccount** [reply]
Created an [issue](https://github.com/isFakeAccount/psnawp/issues/73) for it. I'll try to implement it in version `2.1.0`

**Comment by isFakeAccount** [comment]
@tim140701 v2.1.0 now implements this feature.
